pdr_backend/accuracy/app.py

```diff
@@ -342,12 +352,37 @@ def serve_statistics_from_file():
     If the file cannot be read or another error occurs, it returns a 500 Internal Server Error.
     """
 
+    start_ts = UnixTimeS(int((datetime.utcnow() - timedelta(weeks=4)).timestamp()))
```

Comment:
for better readability, I would separate the date times. I.e.
four_weeks_ago = datetime.utcnow() - timedelta(weeks=4)
start_ts = UnixTimeS(int(four_weeks_ago.timestamp()))

and same for end